src/main/java/de/tum/in/www1/artemis/service/metis/conversation/ChannelService.java

```diff
@@ -330,10 +330,7 @@ public void unarchiveChannel(Long channelId) {
      * @return the created channel
      */
     public Channel createLectureChannel(Lecture lecture, String channelName) {
-        if (!lecture.getCourse().getCourseInformationSharingConfiguration().isMessagingEnabled()) {
-            return null;
-        }
-        Channel channelToCreate = createDefaultChannel(channelName);
+        Channel channelToCreate = createDefaultChannel(channelName != null ? channelName : generateChannelNameFromTitle("lecture-", lecture.getTitle()));
```

Comment:
you could use nullish coalescing here
```suggestion
        Channel channelToCreate = createDefaultChannel(channelName ?? generateChannelNameFromTitle("lecture-", lecture.getTitle()));
```

Reply:
This is java code, so no ;(. See my suggestion with optionals above.

Reply:
The `??` operator doesn't exist in Java. I am using optionals now